aspnetcore/mvc/views/working-with-forms.md

```diff
@@ -583,24 +583,24 @@ The `GetEnumSelectList` method generates a `SelectList` object for an enum.
 
 You can mark your enumerator list with the `Display` attribute to get a richer UI:
 
-[!code-csharp[](working-with-forms/sample/final/ViewModels/CountryEnum.cs?highlight=5,7)]
+[!code-csharp[](working-with-forms/sample/final/ViewModels/CountryEnum.cs?highlight=7,9)]
```

Comment:
Fixes the highlighting:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/e6e969a8-1fd4-4cd8-8bb1-a99c8ccf7513)